packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.push_api.tests.js

```diff
@@ -0,0 +1,361 @@
+import $ from 'jquery';
+import Splitter from 'ui/splitter';
+import fx from 'animation/fx';
+import pointerMock from '../../helpers/pointerMock.js';
+import { isNumeric } from 'core/utils/type';
+import ArrayStore from 'data/array_store';
+import DataSource from 'data/data_source';
+
+import 'generic_light.css!';
+
+const SPLITTER_ITEM_CLASS = 'dx-splitter-item';
+const RESIZE_HANDLE_CLASS = 'dx-resize-handle';
+
+QUnit.testStart(() => {
+    const markup =
+        `<style nonce="qunit-test">
+            #splitterParentContainer {
+                width: 1024px;
+                height: 1024px;
+            }
+        </style>
+        
+        <div id="splitter"></div>
+        <div id="splitterParentContainer">
+            <div id="splitterInContainer"></div>
+        </div>`;
+    $('#qunit-fixture').html(markup);
+});
+
+const moduleConfig = {
```

Comment:
probably its better to move this module to splitter.tests file

Reply:
Yep, it's better. It will allow us to avoid duplication.